### DISCUSSION TITLE
Add default classNames to parts of the table

### DISCUSSION THREAD

**Discussion by GuiPretto** [icflorescu/mantine-datatable#129] [Ideas]
I think adding some default classes to some components would make using css a lot easier. Imagine you want to style the pagination section of the table, since there is no className, and its existence (the pagination section) only display in the tree when demanded, we cant easily reach it using only css.

Using the earlier context, the ideia was to name the pagination section as ```mantine-TablePagination-root``` or something like this.

**Comment by GuiPretto** [comment]
Or, since the project name is mantine-datatable, we could use ```matine-datatable-Pagination-root```. I think this is up to you guys.

**Comment by icflorescu** [comment]
This would kind of break the css encapsulation principle that both Mantine and this library are built upon.
Don't the [additional styling](https://icflorescu.github.io/mantine-datatable/examples/additional-styling) examples achieve the same result?

**Comment by GuiPretto** [comment]
The whole reason behind this idea was because I couldn't reach the pagination ```div``` precisely. I'm preparing my table to sometimes use it and sometimes don't. But since it doesn't have his own identifier it gets hard to get it.

This first image is the table without the pagination component.

![Screenshot_3](https://user-images.githubusercontent.com/28988794/208090003-51cea167-24c4-47a7-a649-595027975e02.png)

But in this second image the pagination component appears in the middle of the other ```div```s. Here, it's class is ```mantine-gzzjso```.

![Screenshot_4](https://user-images.githubusercontent.com/28988794/208090194-ab0eb0b9-66d2-4999-aecc-bca4aaa41566.png)

Having a default className for it would make it easier to reference it in the css.
